test/extensions/open_attrs_test.rb

```diff
@@ -0,0 +1,118 @@
+require "minitest_helper"
+
+module CollectionJson
+  class Serializer
+    class TestOpenAttrs < Minitest::Test
+      include TestHelper
+
+      def setup
+        @user = User.new(name: "Carles Jove", email: "hola@carlus.cat")
+        @serializer = empty_serializer_for(@user)
+        @serializer.class.extensions = [:open_attrs]
+        @serializer.class.template = [
+           email: {
+            prompt: "My email",
+            anything: "at all"
+          }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.